test/liquidations.test.ts

```diff
@@ -104,39 +106,295 @@ describe('isBadDebt', () => {
   });
 });
 
-describe('calculateLiquidationPercent', () => {
+describe('calculateLiquidation', () => {
   let userEstimate: PositionsEstimate;
+  let userPositions: Positions;
 
   beforeEach(() => {
     userEstimate = new PositionsEstimate(0, 0, 0, 0, 0, 0, 0, 0, 0);
   });
-  it('should calculate the correct liquidation percent for typical values', () => {
-    userEstimate.totalEffectiveCollateral = 1000;
-    userEstimate.totalEffectiveLiabilities = 1100;
-    userEstimate.totalBorrowed = 1500;
-    userEstimate.totalSupplied = 2000;
-    const result = calculateLiquidationPercent(userEstimate);
-    expect(Number(result)).toBe(56);
+  it('should find auction subset with partial auction', () => {
```

Comment:
we should add a test for the 1 to 1 happy path at both a partial percentage and the full percentage